### PR TITLE
feat(trace-view): Add project icon to trace view

### PR DESCRIPTION
**Pull request by Zylphrex**
This change adds a project icon to each transaction row to indicate the project
they belong to.

# Screenshot

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/performance/traceDetails/styles.tsx [hunk 1]
```diff
@@ -73,3 +73,9 @@ export const StyledIconChevron = styled(IconChevron)`
   width: 7px;
   margin-left: ${space(0.25)};
 `;
+
+export const TransactionBarTitleContent = styled('span')`
+  display: flex;
+  align-items: center;
+  gap: ${space(0.75)};
```
Comment (k-fish) [inline src/sentry/static/sentry/app/views/performance/traceDetails/styles.tsx:80]: I don't think using `gap` with flex is fully supported still, just do a margin on one of the items.

**Review by k-fish — CHANGES_REQUESTED** [review]
Double check, but I'm pretty sure you can't use `gap` since it's not totally compatible with browsers for flex. 

Aside from that, looks fine. 

As an aside, maybe we should look at the line length, dropdown and icon and tighten it a bit, the lines seem quite long and busy now.

File: src/sentry/static/sentry/app/views/performance/traceDetails/styles.tsx (hunk 1 above)
Comment (Zylphrex) [reply src/sentry/static/sentry/app/views/performance/traceDetails/styles.tsx]: Good call, looks like it's not supported in safari [link](https://developer.mozilla.org/en-US/docs/Web/CSS/gap#browser_compatibility). Fixed in f8b82b0

**Review by dashed — APPROVED** [review]

**Review by k-fish — APPROVED** [review]

**Comment by benvinegar** [comment]
Looks awesome.

**Pull request merged**